### ISSUE TITLE
chroot builds fail to copy out binaries

### ISSUE THREAD

**Issue by Randrianasulu** [AppImage/type2-runtime#77]
I tried chroot build, because I use Slackware 15.0 i586 so docker is unsupported here.

First, build need ALPINE_ARCH, documentation in BUILDING wrong on this point.

Second, it failed to produce binaries in "out" when run like this:

env ALPINE_ARCH=x86 scripts/chroot/chroot_build.sh

I think I fixed that, see attach
[0001-Fix-chroot-build-so-it-copies-out-binaries.txt](https://github.com/user-attachments/files/17446881/0001-Fix-chroot-build-so-it-copies-out-binaries.txt)

basically just use "runtime-${appimage_arch}" instead of plain 'runtime' in find lines.
